### ISSUE TITLE
Figure out why checkout of mozilla-central doesn't respect pref

### ISSUE THREAD

**Issue by linclark** [firefox-devtools/original-console-fork#191]
I have a checkout of mozilla-central that I've been using to land things. It is currently not respecting the pref.

I don't think this is an issue with the code. I've gone back to `Bug 1282658. Part 8` and it still isn't respecting the pref in that checkout, even though we have that commit in our repo. Also, the folks from Taipei have been using the console to test their reps stuff.

I don't feel comfortable continuing to commit to mozilla-central with this as it is, so I think we'll need to wait on #99.

**Comment by bgrins** [comment]
What pref?  `devtools.webconsole.new-frontend-enabled` works for me in a local checkout

**Comment by linclark** [comment]
Yes, that's the pref. I'm glad to hear that it's working for you. That's actually what I expect to be the case.

I'm not sure why my checkout of mozilla-central is doing this. Once we get an end-to-end test in, it should be easier to isolate the issue.

**Comment by linclark** [comment]
This is also happening in `fx-team` in another repo for me at the moment. @bgrins, were you on the latest `fx-team` when you tried?

**Comment by bgrins** [comment]
Just checked on fx-team tip with a clean profile and everything is working as expected - open console first time (it's old frontend), flip pref, close and reopen console (it's new frontend)

**Comment by linclark** [comment]
Ok, it's working for me now. I fetched `mozilla-central` again and reset. I'm not sure whether it was something I was doing or that someone else's code was doing. I did notice that the top commit after fetching was `Backed out changeset eb7f310f128d (bug 1274626) for breaking unified builds` along with a whole bunch of other backouts... dunno if this could have any interaction ¯_(ツ)_/¯

Annnnyway, I'm gonna close this out with fingers crossed.
